### PR TITLE
Pd 84 mention allow volume expansion parameter 

### PR DESCRIPTION
**Pull request by andreipope**
@harsh-px @shailvipx when you have a minute.

### CODE REVIEW

File: content/portworx-install-with-kubernetes/storage-operations/kubernetes-storage-101/volumes.md [hunk 1]
```diff
@@ -41,6 +41,10 @@ In above StorageClass,
 
 [This table](/portworx-install-with-kubernetes/storage-operations/create-pvcs/dynamic-provisioning/#using-dynamic-provisioning) lists all parameters that are supported by the Portworx driver.
 
+{{<info>}}
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/storage-operations/kubernetes-storage-101/volumes.md]: it needs to be more than this. Add a subheading and show users exactly how to do it with an example.

**Comment by harsh-px** [comment]
Also fetch upstream and rebase your branch. Other changes are showing up in your PR.

File: content/portworx-install-with-kubernetes/storage-operations/kubernetes-storage-101/volumes.md (hunk 1 above)
Comment (andreipope) [reply content/portworx-install-with-kubernetes/storage-operations/kubernetes-storage-101/volumes.md]: @harsh-px Added the example we already have. Should I add the commands and their outputs as well?

**Comment by andreipope** [comment]
@harsh-px when you have a minute.

**Comment by andreipope** [comment]
@harsh-px please, take a look at the updates.

**Pull request merged**